info.hiergiltdiestfu.aws.neptune.graphml/src/test/java/info/hiergiltdiestfu/aws/neptune/graphml/BackupTest.java

```diff
@@ -0,0 +1,103 @@
+package info.hiergiltdiestfu.aws.neptune.graphml;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.util.Calendar;
+import java.util.Date;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import com.amazonaws.services.s3.model.S3ObjectSummary;
+
+import info.hiergiltdiestfu.aws.neptune.graphml.AWS.AWSBackupEditor;
+/**
+ * 
+ * @author LUNOACK
+ * Test the BackUp if graphml\AWS\AWSBackupEditor.java works and deletes the correct files.
+ */
+public class BackupTest {
+
+	@BeforeAll
+	static void setup() {
+		backup = new AWSBackupEditor();
+		obj = new S3ObjectSummary();
+	}
+	
+	@BeforeEach
+	void setupCalender(){
+		cal = Calendar.getInstance();
```

Comment:
Das möchte bitte nicht variabel sein, sondern ein festes Datum (oder mehrere) darstellen, anhand dessen dann die verschiedenen Fälle getestet werden. Die Idee ist, dass ein Test immer deterministisch ist, also leicht wiederholt werden kann, ohne dass sich in Abhängigkeit von äußeren Einflüssen das Ergebnis ändert. Dann fallen auch die ifs weiter unten in den @Tests weg, der Test muss stattdessen Fälle für alle Zweige dieses ifs bieten.